packages/fast-usdc/test/mocks.ts

```diff
@@ -59,3 +64,11 @@ export const makeTestLogger = (logger: LogFn) => {
 };
 
 export type TestLogger = ReturnType<typeof makeTestLogger>;
+
+export const makeTestFeeConfig = (usdc: Omit<AmountUtils, 'mint'>): FeeConfig =>
+  harden({
+    flat: usdc.units(1),
+    variableRate: makeRatio(2n, usdc.brand, 100n, usdc.brand),
+    maxVariable: usdc.units(100),
+    contractRate: makeRatio(2n, usdc.brand, 10n, usdc.brand),
```

Comment:
The denominator brand defaults to the numerator brand.
And the denominator defaults to `100n`.

```suggestion
    variableRate: makeRatio(2n, usdc.brand),
    maxVariable: usdc.units(100),
    contractRate: makeRatio(20n, usdc.brand),
```

Reply:
Good tip, thanks